lib/LocalEnsemble.js

```diff
@@ -240,15 +245,51 @@ LocalEnsemble.prototype.predict = function (inputData, method, options, cb) {
     } else {
       missingStrategy = constants.LAST_PREDICTION;
     }
+    if ((typeof options !== 'undefined') &&
+        (options.hasOwnProperty('mean'))) {
+      mean = options.mean;
+    } else {
+      mean = false;
+    }
+    if ((typeof options !== 'undefined') &&
+        (options.hasOwnProperty('addUnusedFields'))) {
+      addUnusedFields = options.addUnusedFields;
+    } else {
+      addUnusedFields = false;
+    }
```

Comment:
these things can be written shorter as:

```js
mean = (typeof options !== 'undefined') && options.hasOwnProperty('mean') && mean
```

but if you find your way better, it's okay with me too.